Utils.ps1

```diff
@@ -404,6 +728,13 @@ function Invoke-RSForEachLab {
         $userAr = @()
     }
 
+    # Convert BastionEnabled to a boolean. If BastionEnabled property is not set, defaults to $false 
+    if ($lab.BastionEnabled) {
```

Comment:
This was already part of the config importing code - in import-vmsetting , is it needed here too?